golang/cosmos/x/vbank/vbank.go

```diff
@@ -241,5 +243,7 @@ func (ch portHandler) Receive(ctx *vm.ControllerContext, str string) (ret string
 }
 
 func (am AppModule) PushAction(ctx sdk.Context, action vm.Jsonable) error {
+	ctx = ctx.WithContext(context.WithValue(ctx.Context(), baseapp.TxHashContextKey, "vbank"))
```

Comment:
I'd prefer a value that is even more obviously not a txHash:
```suggestion
	ctx = ctx.WithContext(context.WithValue(ctx.Context(), baseapp.TxHashContextKey, "x/vbank"))
```